lib/jsdom/level2/style.js

```diff
@@ -173,6 +173,13 @@ core.HTMLStyleElement._init = function() {
 
     evaluateStylesheet.call(this, content, this.sheet, this._ownerDocument.URL);
   });
+
+  this.addEventListener('DOMNodeRemovedFromDocument', function() {
```

Comment:
Hm, honestly? I don't even know where to do that. There's nothing in jsdom that does this, it's all using the mutation events so far :(